package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "test:bdd": "cucumber-js --fail-fast --format progress --format-options '{\"colorsEnabled\": true}' test/bdd/ -r test/bdd/steps/"
+    "test:bdd": "cucumber-js --fail-fast --format progress --format-options '{\"colorsEnabled\": true}' test/bdd/ -r test/bdd/steps/",
+    "test:it": "nyc mocha --exit $(find test/modules -name '*.js')"
```

Comment:
Please, pull the latest changes from v6/develop branch because we already set up everything for unit tests (script for executing unit tests and the file path).

Reply:
Pulled latest v6/dev branch and merged